diff -up lokalize-19.12.2/src/filesearch/filesearchtab.cpp.omv~ lokalize-19.12.2/src/filesearch/filesearchtab.cpp
--- lokalize-19.12.2/src/filesearch/filesearchtab.cpp.omv~	2020-02-22 14:30:27.770979537 +0100
+++ lokalize-19.12.2/src/filesearch/filesearchtab.cpp	2020-02-22 14:30:37.853980495 +0100
@@ -446,7 +446,7 @@ FileSearchTab::FileSearchTab(QWidget *pa
     connect(sh, &QShortcut::activated, ui_fileSearchOptions->querySource, QOverload<>::of(&QLineEdit::setFocus));
     setFocusProxy(ui_fileSearchOptions->querySource);
 
-    sh = new QShortcut(Qt::Key_Escape, this, SLOT(stopSearch()), 0, Qt::WidgetWithChildrenShortcut);
+    sh = new QShortcut(Qt::Key_Escape, this, SLOT(stopSearch()), SLOT(stopSearch()), Qt::WidgetWithChildrenShortcut);
 
     QTreeView* view = ui_fileSearchOptions->treeView;
 
diff -up lokalize-19.12.2/src/glossary/glossarywindow.cpp.omv~ lokalize-19.12.2/src/glossary/glossarywindow.cpp
--- lokalize-19.12.2/src/glossary/glossarywindow.cpp.omv~	2020-02-22 14:18:52.225913469 +0100
+++ lokalize-19.12.2/src/glossary/glossarywindow.cpp	2020-02-22 14:29:41.276975121 +0100
@@ -237,7 +237,7 @@ GlossaryWindow::GlossaryWindow(QWidget *
     m_filterEdit->setPlaceholderText(i18n("Quick search..."));
     m_filterEdit->setFocus();
     m_filterEdit->setToolTip(i18nc("@info:tooltip", "Activated by Ctrl+L.") + ' ' + i18nc("@info:tooltip", "Accepts regular expressions"));
-    new QShortcut(Qt::CTRL + Qt::Key_L, this, SLOT(setFocus()), 0, Qt::WidgetWithChildrenShortcut);
+    new QShortcut(Qt::CTRL + Qt::Key_L, this, SLOT(setFocus()), SLOT(setFocus()), Qt::WidgetWithChildrenShortcut);
     connect(m_filterEdit, &QLineEdit::textChanged, m_proxyModel, &GlossaryNS::GlossarySortFilterProxyModel::setFilterRegExp);
 
     layout->addWidget(m_filterEdit);
diff -up lokalize-19.12.2/src/project/projecttab.cpp.omv~ lokalize-19.12.2/src/project/projecttab.cpp
--- lokalize-19.12.2/src/project/projecttab.cpp.omv~	2020-02-22 14:31:48.563987212 +0100
+++ lokalize-19.12.2/src/project/projecttab.cpp	2020-02-22 14:32:03.073988590 +0100
@@ -126,7 +126,7 @@ ProjectTab::ProjectTab(QWidget *parent)
     m_filterEdit->setPlaceholderText(i18n("Quick search..."));
     m_filterEdit->setToolTip(i18nc("@info:tooltip", "Activated by Ctrl+L.") + ' ' + i18nc("@info:tooltip", "Accepts regular expressions"));
     connect(m_filterEdit, &QLineEdit::textChanged, this, &ProjectTab::setFilterRegExp, Qt::QueuedConnection);
-    new QShortcut(Qt::CTRL + Qt::Key_L, this, SLOT(setFocus()), 0, Qt::WidgetWithChildrenShortcut);
+    new QShortcut(Qt::CTRL + Qt::Key_L, this, SLOT(setFocus()), SLOT(setFocus()), Qt::WidgetWithChildrenShortcut);
 
     l->addWidget(m_filterEdit);
     l->addWidget(m_browser);
